examples/sampling/graphbolt/quickstart/gcn.py

```diff
@@ -0,0 +1,134 @@
+"""
+[Semi-Supervised Classification with Graph Convolutional Networks]
+(https://arxiv.org/abs/1609.02907)
+"""
+import dgl.graphbolt as gb
+import dgl.nn as dglnn
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchmetrics.functional as MF
+
+
+############################################################################
+# (HIGHLIGHT) Create a single process dataloader with dgl graphbolt package.
+############################################################################
+def create_dataloader(dateset, itemset, device):
+    # Sample seed nodes from the itemset.
+    datapipe = gb.ItemSampler(itemset, batch_size=8)
+
+    # Sample neighbors for the seed nodes.
+    datapipe = datapipe.sample_neighbor(dataset.graph, fanouts=[4, 2])
```

Comment:
I am not sure whether GCN is usually train on whole graph(full neighbors) instead of sampling(aggregate local neighbors).

Reply:
Pasted the result in the PR description, it works pretty well.